src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(numb1) {
+    return function(numb2) {
+      counter++;
+
+      if (counter > 3 && counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return Number(numb1 + numb2);
```

Comment:
I am not sure, that "Number" is needed here

Reply:
agree, they are already numbers